src/stcal/ramp_fitting/ols_cas22_util.py

```diff
@@ -0,0 +1,140 @@
+"""Utility routines for Mutli-Accum Ramp Fitting
+"""
+import numpy as np
+
+# Read Time in seconds
+#   For Roman, the read time of the detectors is a fixed value and is currently
+#   backed into code. Will need to refactor to consider the more general case.
+#   Used to deconstruct the MultiAccum tables into integration times.
+READ_TIME = 3.04
```

Comment:
Oops, this is Roman specific.  We need to pull this out as an argument.

Reply:
Seems there was some recent discussion of this value, potentially coming from PPS. I am not seeing anything like this in the schema. What is the process to retrieve from PPS and add as a meta value to the Level 1 file?

Reply:
also note: this is already an argument. the above is the default. However, I will move this into romancal and make the argument required.

Reply:
This is the value to use: https://github.com/spacetelescope/rad/blob/main/src/rad/resources/schemas/exposure-1.0.0.yaml#L268
I don't know if we're populating it yet usually.

Reply:
Sorry I missed that you were just using FRAME_TIME as a default.  I agree with you that it makes more sense to have that on the romancal side.